hugo/content/045_contribute/10_code/10-contribution_guide/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Contribution Guide
+remote: https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md
```

Comment:
maybe add the page https://github.com/gardener/documentation/blob/master/security-release-process.md as another remote page to the hugo generated website so that we don't link to the page externally rendered by github